diff --git a/lib/fxb.d.cts b/lib/fxb.d.cts
index ba98cb07aba1fbd55dd1c1066f1aaeaa877accae..cbd6309d87052697fa355319f5cdcf3fef2590b4 100644
--- a/lib/fxb.d.cts
+++ b/lib/fxb.d.cts
@@ -11,7 +11,7 @@ type Expression = unknown;
  * the parent Matcher's internal state so it always reflects the current parser
  * position with no copying or freezing.
  */
-class MatcherView {
+declare class MatcherView {
   readonly separator: string;
 
   /** Check if current path matches an Expression. */
